src/core/light-controller.ts

```diff
@@ -26,6 +27,9 @@ export class LightController extends NotifyBase<LightController> implements ILig
         this._defaultColor = defaultColor;
         this._lights = entity_ids.map(e => GlobalLights.getLightContainer(e));
         this._lightsFeatures = new LightFeaturesCombined(() => this._lights.map(l => l.features));
+        if (_lightGroupEntityId !== undefined) {
```

Comment:
Also please, here and in every other if condition, that you've added, remove the undefined side of comparison.
It will be like `if(lightGroupEntitiyId)...` or `if(this._lightGroup)...`